### PR TITLE
Mat 7584 qrda export failing

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-7584](https://jira.cms.gov/browse/MAT-7584)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [ ] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: service/web_controller.rb [hunk 1]
```diff
@@ -71,7 +71,7 @@ def as_json(*args)
   measure_dto = request.params # Uses the Rack::JSONBodyParser middleware
 
   # Prepare CQM Measure
-  madie_measure = JSON.parse(measure_dto["measure"])
+  madie_measure = JSON.parse(measure_dto["measure"],:max_nesting => false)
```
Comment (adongare) [inline service/web_controller.rb]: do we know what property/field it is failing on? if yes, we can probably set a number that can be max for that property.

Comment (sb-cecilialiu) [reply service/web_controller.rb]: @adongare No we don't, I did set it to 150 and it worked for this measure, but I guess we probably shouldn't make it work for just this measure? That's why I changed the max number to setting the nesting to false. What do you think?

Comment (adongare) [reply service/web_controller.rb]: I don't have a strong opinion. It might be okay I guess. but this becomes a candidate for DoS attacks. @riddhi-desai / @sb-benohe  might have ideas on this. I remember in Bonnie we had a similar issue while storing documents in the database. there we set the max depth to 500

Comment (sb-cecilialiu) [reply service/web_controller.rb]: @adongare Ok I see your point. I can check the depth and set it to a number. We can also discuss this at the meeting. Thanks so much Ashok!!!

Comment (adongare) [reply service/web_controller.rb]: np

**Review by adongare — DISMISSED** [review]

File: service/web_controller.rb (hunk 1 above)
Comment (sb-cecilialiu) [reply service/web_controller.rb]: @adongare I set max to 512 as Ben suggested, thanks again!

**Review by adongare — APPROVED** [review]

File: service/web_controller.rb (hunk 1 above)
Comment (sb-cecilialiu) [reply service/web_controller.rb]: This is changed to: max_nesting: 512 so resolving this conversation.
